lively.freezer/src/resolvers/node.cjs

```diff
@@ -57,11 +58,17 @@ function ensureFileFormat(url) {
 
 // fixme: if we are bundling from a node.js script but targeting the browser, we need to properly resolve /npm- urls
```

Comment:
This has nothing to do with the PR per se, but isn't this always the case, that we bundle from a node script? :thinking:

Reply:
This is an old comment, from the time when nodejs still would not support esm modules out of the box and would require bundling as well in order to be able to bootstrap its `lively.modules` runtime. We can basically remove this comment since it no longer applies.